Enterprise-CMCS/macpro-mdct-mfp
# Reports Showing in WP Dashboard Table

### Description
Reports now display with edit status in the DashboardTable


### Related ticket(s)
https://qmacbis.atlassian.net/jira/software/c/projects/MDCT/boards/232?modal=detail&selectedIssue=MDCT-2860

---
### How to test
1. Go to /wp ( you may need to comment out  wpReport  in AppRoutes.tsx to get access to /wp)
2. Create new report
3. Report should display as edit
4. Select "edit" which should send you to the full report


<img width="1470" alt="Screenshot 2023-09-07 at 3 49 21 PM" src="https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/14514294/a24c0053-4779-4551-8c09-a52dfec5334f">

Added copy icon and disabled button until SAR is ready
<img width="500" alt="Screenshot 2023-09-07 at 3 49 40 PM" src="https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/14514294/5d52793e-c781-4429-a892-9cd3a1134cca">

Populated report in dashboard table ( When status is "In progress", "Start MFP Work Plan" will be disabled)
<img width="1481" alt="Screenshot 2023-09-07 at 3 49 57 PM" src="https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/14514294/74363b71-018f-46c0-8395-967255c07977">

### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
